Gemfile

```diff
@@ -1,76 +1,61 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
-
 ruby '3.1.3'
-
-gem 'dotenv-rails'
```

Comment:
Kindly re-install this gem as it is needed to keep the database passwords in the `.env` file